### DISCUSSION TITLE
Didn't get student developer pack yet

### DISCUSSION THREAD

**Discussion by AISHWARY-droid** [community/community#135145] [GitHub Education] [answered]
### Hi everyone,

It has been 2 days and i didn't get the benefits of students developer pack. When will i get it ?

**Comment by davevad93** [comment]
Hi @AISHWARY-droid ,

The waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AISHWARY-droid was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by AISHWARY-droid** [reply]
Uhh... How long should i wait... If couple of weeks then 14 days something... Isn't it ?

**Reply by davevad93** [reply]
Yes, a couple of weeks at most, but in most cases the benefits are available in less than 1 week.

**Reply by AISHWARY-droid** [reply]
Thank you sir for your response. Well sir i also wanted to know about campus expert program. Will it be open during this August

**Reply by davevad93** [reply, marked as answer]
You're welcome @AISHWARY-droid , yes applications to the Campus Expert program open in February and August and you'll have a full month to apply.

**Reply by AISHWARY-droid** [reply]
Hello sir, today i saw my account and i am happy to tell you 😊 that i got the pack. And thank you so much sir for clearing my doubts. Sir now i have got this developer pack and i am also new to the tech world so where should i start from

**Reply by davevad93** [reply]
Hey @AISHWARY-droid, glad you finally got the pack; there are plenty of offers available, you can explore them [here](https://education.github.com/pack/offers), just scroll down to the `All Offers` section. Considering you are new to this I suggest you to use any of the learning platforms offered by the pack such as `Codedex`, `Educative` or `FrontendMasters`. There you will get access to many courses on programming languages and frameworks such as Python, JavaScript, Node.js, React and so on.
